tajo-docs/src/main/sphinx/storage_plugins/overview.rst

```diff
@@ -1,5 +1,5 @@
 *************************************
-Storage Plugin
+Storage Plugin Overview
```

Comment:
You've added a handler for PostgreSQL. Please update the expression ```we have a plan to support RDBMS storages like Oracle, MySQL, PostgreSQL.``` in this document.